addons/web/static/src/js/core/rpc.js

```diff
@@ -3,7 +3,7 @@ odoo.define('web.rpc', function (require) {
 
 var ajax = require('web.ajax');
 
-return {
+const rpc = {
```

Comment:
I suppose those changes are for clarity ?

Reply:
There is actually an issue where `this` wasn't correctly bound in some cases, I don't remember which but @alexkuhn should know more.

Reply:
No, `web.test_env` uses `buildQuery()` in standalone way (unbound), and we have tests that do RPC with `search_read`, and it crashed in line `this._serializeSort` due to `this` being `undefined`.